lte/gateway/python/integ_tests/defs.mk

```diff
@@ -132,6 +132,11 @@ s1aptests/test_service_req_ul_udp_data_with_mme_restart.py \
 s1aptests/test_attach_detach_setsessionrules_tcp_data.py \
 s1aptests/test_enable_ipv6_iface.py \
 s1aptests/test_ipv6_non_nat_dp_ul_tcp.py \
+s1aptests/test_ipv4v6_non_nat_ul_tcp.py \
```

Comment:
If these test cases are made part of sanity, please remove it from the commented segment in line numbers: 284-290

Reply:
Oh I think I checked-in these changes by mistake. I am still checking if they are stable.